api/v1beta2/foundationdbcluster_types.go

```diff
@@ -943,6 +1001,29 @@ type MaintenanceModeOptions struct {
 	MaintenanceModeTimeSeconds *int `json:"maintenanceModeTimeSeconds,omitempty"`
 }
 
+// TaintReplacementOption defines the taint key and taint duration the operator will react to a tainted node
+// Example of TaintReplacementOption
+//   - key: "example.org/maintenance"
+//     durationInSeconds: 7200 # Ensure the taint is present for at least 2 hours before replacing Pods on a node with this taint. -1 disable the handling of this tainted key
+//   - key: "*" # The wildcard would allow to define a catch all configuration
+//     durationInSeconds: 3600 # Ensure the taint is present for at least 1 hour before replacing Pods on a node with this taint
+//
+// Setting key as * and its durationInSeconds as negative integer will disable the entire taint feature and override taint options on other keys
+//   - key: "*"
+//     durationInSeconds: -1
```

Comment:
I thought we removed the feature to disable the taint feature with `*` set to a negative number?

Reply:
changed it to 


// Setting durationInSeconds to the maximum of int64 will practically disable the taint key.
// When a Node taint key matches both an exact TaintReplacementOption key and a wildcard key, the exact matched key will be used.